app/jobs/hyku_addons/reindex_model_job.rb

```diff
@@ -35,7 +34,7 @@ def perform(klass, cname, limit: 25, page: 1, options: {})
     private
 
       def mint_doi(work)
-        return if can_mint_for?(work)
+        return unless cant_mint_for(work)
```

Comment:
`unless cant` is a double negative and should be avoided.  The method is named negatively, and the whole condition at the end is negated.  This is even more confusing than before.

Reply:
Thanks we might need to pack the pr or may be revert to the original since this current changes was to try and address what you said was confusing. 

I will revert to the original and push and it is still confusing, I am happy to throw away the PR and leave the job as it is for now and return to it later.

Reply:
This is entirely fixable.  Please do this:
````
def can_mint_for?(work)
  -- conditions under which a work can be minted 
end
```

Then call it in the way you want to test for... if something can be minted, or unless it can be minted.  I don't understand why this is becoming so difficult.  Please do not revert because it is now unclear in your original code whether you had the method named wrongly all along.